### PR TITLE
hw done

### PR DESCRIPTION
**Pull request by anfaljay**
### How difficult did you find this (0 - 10)? (0 very easy, 10 very difficult)
7

### How long did it take?
too long

### Was there anything that you struggled with?  If so, what?
searching for methods in google

### Is there anything that you would like some further information on?
best way to solve it. maybe I used inconvenient ways.

### Do you have any suggestions to improve this assignment?
Hints? helpful links rather than a general one.

### CODE REVIEW

File: data-collections-exercises.md [hunk 1]
```diff
@@ -9,89 +9,147 @@ planeteers = ["Earth", "Wind", "Captain Planet", "Fire", "Water"]
...
+planeteers.push"Heart"
+=> ["Earth", "Wind", "Captain Planet", "Fire", "Water", "Heart"]
 ```
 
 Remove "Captain Planet" from the array (without using a method).
 
 ```rb
-# Your answer here
+planeteers.delete_at(2)
+=> "Captain Planet"
```
Comment (ghadeer-x) [inline data-collections-exercises.md:27]: you used a method we need you to know the simple cool trick that Ruby provide which is the subtraction `planeteers. - ["Captain Planet"]`

File: data-collections-exercises.md [hunk 2]
```diff
@@ -9,89 +9,147 @@ planeteers = ["Earth", "Wind", "Captain Planet", "Fire", "Water"]
...
+heroes.shuffle.first
+=> "Fire"
 ```
 
 Select all elements in `heroes` that begin with "B" using a method. [The Ruby documentation might help](http://ruby-doc.org/core-2.6.1/Array.html).
 
 ```rb
-# Your answer here
+# def with_b(heroes, B)
+#   heroes.select { |str| str.start_with?(B) }
```
Comment (ghadeer-x) [inline data-collections-exercises.md:85]: you forgot the quotation`""` heroes.select { |str| str.start_with?("B") }

File: data-collections-exercises.md (hunk 1 above)
Comment (anfaljay) [reply data-collections-exercises.md:27]: Nice! much easier. Thank you

**Comment by ghadeer-x** [comment]
with time and practice, you will solve it much faster !! Great Job Anfal !! 👏

File: data-collections-exercises.md [hunk 3]
```diff
@@ -9,89 +9,147 @@ planeteers = ["Earth", "Wind", "Captain Planet", "Fire", "Water"]
...
 ### Hashes
 
 Initialize a hash called `ninja_turtle` with the properties `name`, `weapon` and `radical`. They should have values of "Michelangelo", "Nunchuks" and `true` respectively.
 
 ```rb
-# Your answer here
+ninja_turtle = {
+    name: "Michelangelo",
+    weapon: "Nunchuks",
+    radical: `true`}
```
Comment (ghadeer-x) [inline data-collections-exercises.md:97]: true is a boolean you don't need ` backtick
```
ninja_turtle = {
    name: "Michelangelo",
    weapon: "Nunchuks",
    radical: true 
}
```

File: data-collections-exercises.md (hunk 1 above)
Comment (ghadeer-x) [reply data-collections-exercises.md:27]: you're welcome ⭐️

**Pull request closed without merging**